spec/presenters/menu/custom_loader_spec.rb

```diff
@@ -4,10 +4,21 @@
     Singleton.__init__(Menu::CustomLoader)
   end
 
+  def register(item)
+    engine = Object.new.tap do |engine|
+      allow(engine).to receive(:menu).and_return([ item ])
+    end
+
+    allow(Vmdb::Plugins).to receive(:all).and_return([ engine ])
+
+    Menu::CustomLoader.unload
+    Menu::Manager.instance.send(:initialize)
+  end
+
   context '.register' do
```

Comment:
You probably should add an `after` block here that calls `Menu::CustomLoader.unload` (and some sort of cleanup for Menu::Manager) in order to put the test suite back into a clean state